sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTest.java

```diff
@@ -292,6 +417,22 @@ public void processElement(@SuppressWarnings("unused") ProcessContext c) {
   }
 
   @Test
+  public void testProcessElementExceptionWithReturn() throws Exception {
```

Comment:
I don't think we use type of input and output of the DoFn as type of local variable, anything on the stack, or return type in generated code. Why do we need such a test?